kolibri/content/api.py

```diff
@@ -29,12 +32,10 @@ def title_description_filter(self, queryset, value):
         exact_match = queryset.filter(Q(parent__isnull=False), Q(title__icontains=value) | Q(description__icontains=value))
         if exact_match:
             return exact_match
-        # if no exact match, search for non-adjacent match
-
+        # if no exact match, fuzzy search using the stemmed_metaphone field in ContentNode that covers the title and description
         return queryset.filter(
             Q(parent__isnull=False),
-            reduce(lambda x, y: x & y, [Q(title__icontains=word) for word in value.split()]) |
-            reduce(lambda x, y: x & y, [Q(description__icontains=word) for word in value.split()]))
+            reduce(lambda x, y: x & y, [Q(stemmed_metaphone__icontains=doublemetaphone(self.stemmer.stem(word))[0]) for word in value.split()]))
```

Comment:
lookup both hashes enabled. good to go!
the CC end change is here https://github.com/fle-internal/content-curation/pull/76/commits/14c327d4dde8a595e4bd4d6e97e9c7c4ca7d1a17